src/testnet/helium/event_dispatcher.rs

```diff
@@ -0,0 +1,242 @@
+                    "txid": format!("0x{:?}", txid),
+                    "type": "contract_event",
+                    "contract_event": {
+                        "contract_identifier": event_data.key.0.to_string(),
+                        "topic": event_data.key.1,
+                        "value": event_data.value,
+                    }
+                }),
+                StacksTransactionEvent::STXEvent(STXEventType::STXTransferEvent(event_data)) => json!({
+                    "txid": format!("0x{:?}", txid),
+                    "type": "stx_transfer_event",
+                    "stx_transfer_event": event_data
+                }),
+                StacksTransactionEvent::STXEvent(STXEventType::STXMintEvent(event_data)) => json!({
+                    "txid": format!("0x{:?}", txid),
+                    "type": "stx_mint_event",
+                    "stx_mint_event": event_data
+                }),
+                StacksTransactionEvent::STXEvent(STXEventType::STXBurnEvent(event_data)) => json!({
+                    "txid": format!("0x{:?}", txid),
+                    "type": "stx_burn_event",
+                    "stx_burn_event": event_data
+                }),
+                StacksTransactionEvent::NFTEvent(NFTEventType::NFTTransferEvent(event_data)) => json!({
+                    "txid": format!("0x{:?}", txid),
+                    "type": "nft_transfer_event",
+                    "nft_transfer_event": event_data
+                }),
+                StacksTransactionEvent::NFTEvent(NFTEventType::NFTMintEvent(event_data)) => json!({
+                    "txid": format!("0x{:?}", txid),
+                    "type": "nft_mint_event",
+                    "nft_mint_event": event_data
+                }),
+                StacksTransactionEvent::FTEvent(FTEventType::FTTransferEvent(event_data)) => json!({
+                    "txid": format!("0x{:?}", txid),
+                    "type": "ft_transfer_event",
+                    "ft_transfer_event": event_data
+                }),
+                StacksTransactionEvent::FTEvent(FTEventType::FTMintEvent(event_data)) => json!({
+                    "txid": format!("0x{:?}", txid),
+                    "type": "ft_mint_event",
+                    "ft_mint_event": event_data
+                }),
+            }).collect();
+
+        let serialized_txs: Vec<serde_json::Value> = receipts.iter().map(|artifact| {
+            
+            let response_data = match &artifact.result {
+                Value::Response(data) => data,
+                _ => unreachable!(),
+            };
+
+            let tx = &artifact.transaction;
+            let mut unsigned_tx_bytes = vec![];
+            tx.consensus_serialize(&mut unsigned_tx_bytes).unwrap();
+            let formatted_bytes: Vec<String> = unsigned_tx_bytes.iter().map(|b| format!("{:02x}", b)).collect();
+            json!({
+                "txid": format!("0x{}", tx.txid()),
```

Comment:
Including `tx_index` (u32?) would be useful here

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/1319/commits/7bc35a9b5b94b659a899410f6b1057f0dfa8ad07